### PR TITLE
Updated API to now create machine sets for workers

### PR DESCRIPTION
**Pull request by zachpuck**
Any Worker pool will becoming a MachineSet CR. I also updated the API to accept count as an integer instead of string.

### CODE REVIEW

File: internal/apiserver/cluster.go [hunk 1]
```diff
@@ -81,29 +81,42 @@ func (s *Server) CreateCluster(ctx context.Context, in *pb.CreateClusterMsg) (*p
...
+				Replicas: int(machineSetConfig.Count),
+				Selector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"node-pool": machineSetConfig.Name,
+					},
+				},
+				MachineTemplate: v1alpha.MachineTemplate{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: machineLabels,
+					},
```
Comment (oneilcin) [inline internal/apiserver/cluster.go]: You need to add the Selector label as one of the MachineTemplate labels.  It will not validate without this.

Comment (oneilcin) [reply internal/apiserver/cluster.go]: hmmm, or should we leave that up to the user??

Comment (davidewatson) [reply internal/apiserver/cluster.go]: For the gRPC interface I don't think we expose k8s-isms if we can avoid it.

Comment (zachpuck) [reply internal/apiserver/cluster.go]: I add the selector label at [line 90](https://github.com/samsung-cnct/cma-ssh/blob/b4f68e8b95337e38211892e8d810ecd710c440a7/internal/apiserver/cluster.go#L90), it becomes part of the machineSet labels and each of the machine labels.

**Comment by zachpuck** [comment]
Updated cma-ssh e2e test an ran with results:
`full-test-cma-ssh PASSED`

I moved the machineSet creation to be the first thing that is done when creating a cluster, I felt this would help block the control plane from being created if the machineSet failed to validate.

**Pull request merged**